pulpcore/tasking/worker.py

```diff
@@ -232,7 +241,11 @@ def identify_unblocked_tasks(self):
         taken_exclusive_resources = set()
         taken_shared_resources = set()
         # When batching this query, be sure to use "pulp_created" as a cursor
-        for task in Task.objects.filter(state__in=TASK_INCOMPLETE_STATES).order_by("pulp_created"):
+        for task in (
+            Task.objects.filter(state__in=TASK_INCOMPLETE_STATES)
+            .order_by("pulp_created")
+            .select_related("pulp_domain")
```

Comment:
Would that work too?
```suggestion
            .select_related("pulp_domain__name")
```

Reply:
It would, if `name` was a ForeignKey.

Reply:
So no? That's fine.